### DISCUSSION TITLE
Game Downloader or FreeStore

### DISCUSSION THREAD

**Discussion by aenrione** [XboxUnity/AuroraScripts#42] [Q&A] [answered]
Hello!

Pardon if my question comes off as a newbie approach since i don't know the toolset of Aurora and I just began experimenting with homebrew.

I was thinking of creating a script that retrieved a list of games from **MyRient** roms and then be able to download the .zip files (like a Game Downloader or Freestore).

The idea would be to make the script download and extract some zip, and then create an external tool (maybe a .xex app) that replicated the _extract-xiso_ functionality so we could download games from the console itself.

I have messed around with a script that retrieved the game list from the html but i am failing at both:
1. downloading the large zip
2. Extracting a large zip

Even though my script technically works for small sample zip files.

Is this task even achievable? What would it take to implement something like this.

Thanks a lot!

**Comment by Swizzy** [comment, marked as answer]
Unfortunately this is borderline illegal, it 100% enables and promotes piracy which we cannot provide help around.

That said, you could make it all into a simple xex since you would need that anyway.

I would recommend starting of with a simple console style application (text only), you can find examples in my repos among other places, it is possible to then highlight current selection with text etc.

It won't be as pretty but is a good way to get started and make the logic work, it could then be refined with a nicer UI
